grpc/conn.go

```diff
@@ -16,6 +16,8 @@ type ReconfigurableClientConn struct {
 	conn   rpc.ClientConn
 }
 
+var errNotConnected = errors.New("not connected")
```

Comment:
Should this be public so callers can test for it?

Reply:
I'm inclined to leave this private. If I were to make this public, I suspect it'd be some false promise that this would be getting returned for everything someone cares about. I'd need to find other places to substitute in the constant.

I'll add a comment justifying why* this is a constant.